services/wallet/router/pathprocessor/processor_bridge_celar.go

```diff
@@ -66,6 +67,10 @@ func (s *CelerBridgeProcessor) Name() string {
 	return ProcessorBridgeCelerName
 }
 
+func (s *CelerBridgeProcessor) Clear() {
+	// no need for clearing prodTransferConfig and testTransferConfig cause they don't depend on input params
+}
+
```

Comment:
I notice that only two structs actually has logic in their `Clear()` interface method implementation. Is it possible that this function could be part of a separate interface?

Something like the below:

```go
// Define a separate interface for the Clear method
type Clearable interface {
    Clear()
}
```

Then in the logic you can call like below, meaning only the structs that need to call `Clear()` implement this function.

```go
    if clearable, ok := c.(Clearable); ok {
        clearable.Clear()
    }
```

Reply:
Done, thanks!